packages/desktop/views/dashboard/vesting/Vesting.svelte

```diff
@@ -31,8 +35,14 @@
         MeatballMenuButton,
         TooltipIcon,
     } from '@ui'
+    import { onMount } from 'svelte'
 
     const DEFAULT_EMPTY_VALUE_STRING = '----'
+    let modal: Modal
+    let timeUntilNextPayout = DEFAULT_EMPTY_VALUE_STRING
+    let minRequiredStorageDeposit: number
+    const availableOuputs = undefined
```

Comment:
this variable is always `undefined`. I think that before you updated it in `areOutputsReadyForConsolidation`, because the boolean that the promise returns is not being used